apps/prairielearn/src/tests/groupExam.test.ts

```diff
@@ -66,7 +67,7 @@ async function switchUserAndLoadAssessment(
   const $ = cheerio.load(page);
 
   // Check that the correct CSRF form exists
-  const elementQuery = `form[name="${formName}"] input[name="__csrf_token"]`;
+  const elementQuery = `form${formName ? `[name="${formName}"]` : ''} input[name="__csrf_token"]`;
```

Comment:
Why this change? When would we want to grab *any* CSRF token on the page? This makes this test less useful as it doesn't actually assert that the form we're submitting has a CSRF token in it.

Reply:
This change was necessary because the leave group form is now using a modal form, which doesn't have a name. I have updated it in abcb6d8d3 to use a container instead in this case, so that instead of finding a form from the name, you find the form inside the modal.